frontend/src/queries/schema/schema-assistant-messages.ts

```diff
@@ -84,12 +85,26 @@ export interface ReasoningMessage extends BaseAssistantMessage {
     substeps?: string[]
 }
 
+/**
+ * The union type with all cleaned queries for the assistant. Only used for generating the schemas with an LLM.
+ */
+export type AssistantQueryType =
+    | AssistantTrendsQuery
+    | AssistantFunnelsQuery
+    | AssistantRetentionQuery
+    | AssistantHogQLQuery
+
+/**
+ * The union type with all supported base queries for the assistant.
+ */
+export type SupportedAssistantQuery = TrendsQuery | FunnelsQuery | RetentionQuery | HogQLQuery
```

Comment:
Naming nit: These sound almost the same. Maybe something like this:
`AssistantQueryType` →  `AnyAssistantGeneratedQuery`
`SupportedAssistantQuery` →  `AnyAssistantSupportedQuery`